website/components/LandingPage/index.tsx

```diff
@@ -396,6 +404,16 @@ export default function LandingPage(): ReactElement {
       <ContentContainer style={{ paddingBottom: "400px" }}>
         <ProjectList>{landingPageContent.map(renderProject)}</ProjectList>
       </ContentContainer>
+
+      <ContentContainer style={{ padding: "0 30px 40px 30px" }}>
+        <Divider />
+        <FlexColumnAlignCenter style={{ paddingTop: "20px" }}>
+          <CookieSettingsButton type="text" className="ot-sdk-show-settings">
+            Cookie settings
+            <VisuallyHidden>(opens popup menu)</VisuallyHidden>
```

Comment:
Is the <VisuallyHidden> part for accessibility? I don't write front end code very much, but this seems very handy!